controllers/API/comment.js

```diff
@@ -0,0 +1,97 @@
+var express = require('express');
+var router  = express.Router();
+var Comment = require('mongoose').model('comment');
+var Forum = require('mongoose').model('forum');
+
```

Comment:
This is indented with spaces. My IDE uses tabs and that would result in unwanted merge conflicts again. This issue should be solved once we introduce `editorconfig` and insert all the styling rules we'd like.

Reply:
Yes lets do that and get rid of this problem for good.